site/about/overview-model-risk-management.qmd

```diff
@@ -14,9 +14,11 @@ Overview & graphics
 
 Our ValidMind model risk management platform offers an integrated platform to manage validation reports, track findings, and report on model risk compliance across your model portfolio. Its main purpose is to enable your organization to monitor and manage models effectively, focusing on mitigating risks, maintaining governance, and ensuring compliance throughout the entire enterprise.
 
-## The ValidMind AI Risk Platform 
+## The {{< var vm_risk >}} 
```

Comment:
GLOBAL COMMENT: There's a couple more variations of product names that need to be fixed:

- "ValidMind model risk management platform", 2 occurrences but only this file here
- "ValidMind Platform" — 44 occurrences in 29 files, see search results 

**"ValidMind Platform", see line 27 for example**

```
44 results - 29 files

site/index.qmd:
  104                      <h2 style="color: var(--bs-white);">Support &amp; Training</h2>
  105:                     <p style="margin-bottom: 16px;color: var(--bs-white);">You can learn more about effective model risk management by requesting a demo with the ValidMind Platform.</p>
  106                      <p style="margin-bottom: 28px;color: var(--bs-gray-400);padding: 16px;background: transparent;border-radius: 0;padding-top: 0;padding-bottom: 0;padding-right: 0;border: 1px none var(--bs-white);border-left-style: solid;border-left-color: var(--bs-gray-400);"><strong>Need some help? Try our self-service documentation or email us at:&nbsp;</strong><a href="mailto:support@validmind.com"><strong><span style="text-decoration: underline;">support@validmind.com</span></strong></a></p><a class="btn btn-secondary" role="button" style="padding: 12px;color: var(--bs-white);background: transparent;border-radius: 4px;margin-top: 4px;font-family: 'Inter';font-style: normal;font-weight: 700;font-size: 14px;line-height: 100%;margin-right: 8px;border: 1px solid var(--bs-white);border-left-color: var(--bs-white);" href="guide/faq.html">Frequently Asked Questions</a><a class="btn btn-primary" role="button" style="padding: 12px;color: var(--bs-white);background: #DE257E;border-radius: 4px;margin-top: 8px;font-family: 'Inter';font-style: normal;font-weight: 700;font-size: 14px;line-height: 100%;border: 1px solid var(--bs-pink);" href="https://validmind.com/demo/">Request A Demo</a>

site/about/overview-model-risk-management.qmd:
  26  
  27: With the ValidMind Platform, you can:
  28  

site/about/overview.qmd:
  44  ::: {.g-col-6}
  45: ![](../assets/img/Group 38-2-compact.png){width=85% fig-alt="An image showing the main ValidMind Platform UI main dashboard"}
  46  :::

site/guide/collaborate-with-others.qmd:
  18  
  19: - You are [logged into the ValidMind Platform](login.qmd)
  20  - You hold the `Developer` or `Validator` role

site/guide/customize-documentation-templates.qmd:
  13  
  14: The template you want to edit must have been added to the ValidMind Platform already.
  15  

site/guide/export-documentation.qmd:
  12  * Model documentation is completed or in progress
  13: * You are logged into the ValidMind Platform
  14  

site/guide/get-started-developer-framework.qmd:
  42  
  43: ![](get-started-validmind.png){width=70% fig-align="center" fig-alt="An image showing the two main components of ValidMind. The developer framework that integrates with your existing developer environment, and the ValidMind Platform UI."}
  44  

site/guide/get-started-sandbox.qmd:
  115  <form method="get" action="https://app.prod.validmind.ai/" target="_blank">
  116:    <button type="submit" style="color: white; background-color: #de257e; border-radius: 8px; border: none; font-size: 16px; padding: 6.25px 12.5px; margin-left: 16px; margin-bottom: 20px;">ValidMind Platform UI</button>
  117  </form>

site/guide/get-started.qmd:
  17  
  18: ![](get-started-validmind-lifecycle.jpg){width=70% fig-align="center" fig-alt="An image showing the two main components of ValidMind. The developer framework that integrates with your existing developer environment, and the ValidMind Platform UI."}
  19  

site/guide/install-and-initialize-developer-framework.qmd:
   9  
  10: In order to integrate the {{< var vm_dev >}} and to be able to upload to the ValidMind Platform, you must provide the following information through a code snippet that you copy from the {{< var vm_platform >}}:
  11  

  20  
  21: The {{< var vm_dev >}} also requires access to the data sources where data sets used for training, testing, and trained model files are stored. This access is needed to run model documentation and validation tests, and to upload to the ValidMind Platform to populate the model documentation and validation reports.
  22  

  68  
  69: After you have pasted the code snippet into your development source code and run your code, the Python client library will connect and register with ValidMind. You can now use the {{< var vm_dev >}} to document and test your models, and to upload model documentation and test results to the ValidMind Platform.

site/guide/login.qmd:
   9  
  10: - A valid email address registered with the ValidMind Platform.
  11  - Your password associated with the registered email address.

  25  
  26: After successful login, you are redirected to the main dashboard of the ValidMind UI where you can start exploring the features of the ValidMind Platform.
  27  

site/guide/next-steps.qmd:
  7  
  8: - [**For platform administrators**](guides.qmd#onboarding) — Learn how to configure the platform, from onboarding new users to setting up connectivity via AWS PrivateLink, to customizing the ValidMind Platform to suit your existing workflows, and more. 
  9  

site/guide/quickstart-explore-sample-model-documentation.qmd:
   9  
  10: ![](quickstart-platform-dashboard.png){width=90% fig-alt="An image showing the ValidMind Platform UI main dashboard"}
  11  

  19  
  20: To try out ValidMind, you need to be a registered user on the ValidMind Platform.
  21  

  74   
  75: Continue with [Register your first model](quickstart-register-your-first-model.qmd) to learn more about using the ValidMind Platform hands-on.

site/guide/quickstart-generate-documentation-for-your-model.qmd:
   9  
  10: To try out ValidMind, you need to be a registered user on the ValidMind Platform.
  11  

  29  
  30:    This code snippet can be copied and pasted directly into your developer source code to integrate the {{< var vm_framework >}} and to be able to upload to the ValidMind Platform. 
  31     

  51     
  52:    The notebook will install the ValidMind library and then connect to the ValidMind Platform.
  53  

site/guide/quickstart-register-your-first-model.qmd:
   5  
   6: To be able to document models with the ValidMind Platform, you need to register them in the model inventory first. Let's show you how. 
   7  

   9  
  10: To try out ValidMind, you need to be a registered user on the ValidMind Platform.
  11  

site/guide/quickstart-try-developer-framework-in-your-own-developer-environment.qmd:
   9  
  10: To try out ValidMind, you need to be a registered user on the ValidMind Platform.
  11  

  37     
  38:    The notebook will guide you through installing the {{< var vm_framework >}}, initializing the Python environment, and finally initializing the ValidMind Client Library by connecting to the ValidMind Platform.
  39     

  64     
  65:    The notebook will guide you through installing the {{< var vm_framework >}}, initializing the Python environment, and finally initializing the ValidMind Client Library by connecting to the ValidMind Platform.
  66     

site/guide/quickstart-try-developer-framework-with-colab.qmd:
   9  
  10: To try out ValidMind, you need to be a registered user on the ValidMind Platform.
  11  

site/guide/quickstart-try-developer-framework-with-jupyterhub.qmd:
  15  
  16: To try out ValidMind, you need to be a registered user on the ValidMind Platform.
  17  

  44     
  45:    The notebook will guide you through installing the {{< var vm_framework >}}, initializing the Python environment, and finally initializing the ValidMind Client Library by connecting to the ValidMind Platform.
  46     

site/guide/quickstart.qmd:
   9  
  10: To try out ValidMind, you need to be a registered user on the ValidMind Platform.
  11  

  28  
  29: To upload from the {{< var vm_dev >}} to the ValidMind Platform and to access the web user interface, you must be able to access our domains:
  30  

site/guide/release-notes-2023-may-30.qmd:
  18  
  19: - You can now export model documentation to Word documents from the {{< var vm_ui >}}. This feature enables you make use of model documentation or validation report files outside of the ValidMind Platform. [Learn more ...](export-documentation.qmd)
  20  

site/guide/support.qmd:
  5  
  6: Our support team can provide you with quick and easy access to the resources you need to troubleshoot technical issues and help you get the most out of the ValidMind Platform.
  7  

site/guide/troubleshooting.qmd:
  31  
  32: Follow the steps in [Install and initialize the {{< var vm_dev >}}](install-and-initialize-developer-framework.qmd) for detailed instructions on how to integrate the {{< var vm_dev >}} and upload to the ValidMind Platform.
  33  

site/guide/view-documentation-templates.qmd:
   5  
   6: Learn how to view the structure and configuration of existing documentation templates on the ValidMind Platform. 
   7  

   9  
  10: To view templates, you must be a registered user on the ValidMind Platform.
  11  

site/guide/view-validation-guidelines.qmd:
  12  - Model documentation has already been submitted for review by the model validation team for this project
  13: - You are logged into the ValidMind Platform
  14  

site/releases/2024-jan-18/highlights.qmd:
  24  ::: g-col-6
  25: ![Light mode](light-mode.png){fig-alt="The ValidMind Platform UI in light mode"}
  26  :::

  28  ::: g-col-6
  29: ![Dark mode](dark-mode.png){fig-alt="The ValidMind Platform UI in dark mode"}
  30  :::

site-unused/before-you-begin.qmd:
  13  
  14: To try out ValidMind, you need to be a registered user on the ValidMind Platform.
  15  
  16: To connect the {{< var vm_framework >}} to the ValidMind Platform and to access the web user interface, you must be able to access our domains:
  17  

site-unused/create-documentation.qmd:
  43  
  44: The code snippet can be copied and pasted directly into your developer source code to integrate the {{< var vm_framework >}} and to be able to upload to the ValidMind Platform.
  45  

site-unused/mermaid-diagrams-dev-framework.qmd:
  21      E -- No --> H[Create and register additional tests]
  22:     G --> I[ValidMind Platform UI:<br>Add content blocks to model documentation]
  23      H --> E

site-unused/review-documentation.qmd:
  15  * A model developer has started generating documentation, either using the {{< var vm_dev >}} or via the online UI editor
  16: * You are logged into the ValidMind Platform
  17  
```

Reply:
Ahhh I didn't even think of the other variations of this, sorry. Did you end up making the changes or should I adjust?

Reply:
Same here, the conversation is marked as resolved but no changes have been committed to address this comment.